packages/terra-consumer-icon/tests/nightwatch/icon-spec.js

```diff
@@ -1,16 +1,16 @@
 /* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
 // eslint-disable-next-line import/no-extraneous-dependencies
-const screenshot = require('terra-toolkit').screenshot;
+// const screenshot = require('terra-toolkit').screenshot; // TODO: replace this with wdio/Terra screenshots?
 
 module.exports = {
   before: (browser, done) => {
     const size = browser.globals.breakpoints.medium;
     browser.resizeWindow(size[0], size[1], done);
   },
 
-  afterEach: (browser, done) => {
-    screenshot(browser, 'terra-consumer-icon', done);
-  },
+  // afterEach: (browser, done) => {
+  //   screenshot(browser, 'terra-consumer-icon', done); // TODO: replace this with wdio/Terra screenshots?
+  // },
```

Comment:
Is this needed anymore / should this be replaced with Terra toolkit screenshots like [terra-button wdio tests](https://github.com/cerner/terra-core/blob/master/packages/terra-button/tests/wdio/button-spec.js#L9)?

Reply:
we need to add wdio test, but that's not in the scope of this task

Reply:
Yeah, I initially had tried adding it, but it seemed to add _a lot_ of complexity to this JIRA. In the end, I ended up reverting the changes.

Reply:
nightwatch tests fail if left in

todo was left in intentionally to get feedback during the PR :)

Reply:
lets save wdio tests for another PR later. It will take more effort to setup the CI for it as well I believe